<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxPersonal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>611</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(220, 215, 255);</string>
    </property>
    <property name="title">
     <string>Personal Information: </string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>541</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fullNameLabel">
        <property name="text">
         <string>Full Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fullNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="labelGender">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_Male">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFemale">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxOther">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>621</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(210, 255, 199);</string>
    </property>
    <property name="title">
     <string>Other Information</string>
    </property>
    <widget class="QLabel" name="labelAddress">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAddress">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEducation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Education:</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>160</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonBachelor">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Bachelor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonMaster">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Master</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDoctoral">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Doctoral</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSendData">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
